pom.xml

```diff
@@ -1173,7 +1173,7 @@
     </profile>
     <!-- See BUILDING.md for profile selection -->
     <profile>
-      <id>phoenix-hbase-compat-1.3.0-default</id>
+      <id>phoenix-hbase-compat-1.5.0-default</id>
```

Comment:
Why did we change this?

Reply:
With 1.3 as the default, it's difficult to run the 1.5-specific tests like WALAnnotationIT in IntelliJ, and it means they won't get exercised in the automated test runs. FYI, @stoty .

Reply:
Since 4.x multibranch keep running all tests with all supported versions including 1.5, pom.xml changes will be reverted before merging this PR?